samples/v2/pipeline_with_env.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from kfp import compiler
+from kfp import components
+from kfp import dsl
+from kfp.dsl import component
+
+
+@component
+def print_env_op():
+    import os
+    print('ENV1', os.environ.get('ENV1'))
+    print('ENV2', os.environ.get('ENV2'))
+
+
+print_env_2_op = components.load_component_from_text("""
+name: Check env
+implementation:
+  container:
+    image: alpine
+    command:
+    - sh
+    - -c
+    - |
+      set -e -x
+      if [ "$ENV2" == "val2" ]; tjhen
```

Comment:
typo: then

This is also an indicator the test wasn't even run.
You need to add to the list: https://github.com/kubeflow/pipelines/blob/d17916fc6a6c9ecee322466105561c0feb99662d/samples/test/config.yaml#L118-L128

Reply:
Thank you, just fixed this.